CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - On macOS, fix `CursorMoved` event reporting the cursor position using logical coordinates.
 - On macOS, fix issue where unbundled applications would sometimes open without being focused.
 - On macOS, fix `run_return` does not return unless it receives a message.
+- On macOS, updated core-* dependencies and cocoa
```

Comment:
This was added under 0.21.0 rather than unreleased.

Reply:
How do I fix this?

Reply:
Change it in a new PR based on master I guess.